recipes/kernax/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: kernax
+  version: 0.0.1
+
+source:
+  # git_url: https://gitlab.com/drti/kernax.git
+  url: https://gitlab.com/drti/kernax/-/archive/main/kernax-main.tar.gz
+  sha256: bd3896bbb15120ebc3c7732899451f924dba8367d1b1119a61018295664f8fd1
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]  
+  number: 0
+
+requirements:
+  build:
```

Comment:
This should be `host` instead of `build`. Also, I believe `setuptools-scm` is missing (it's in the `[build-system]` upstream).

Reply:
Fixed, thank you.